doc/source/cluster/cloud.rst

```diff
@@ -146,6 +146,31 @@ Ray with cloud providers
             # Tear down the cluster
             $ ray down ray/python/ray/autoscaler/staroid/example-full.yaml
 
+    .. group-tab:: Aliyun
+
+        First, you need to configure your aliyun account. You need to set access key and access key secret in yaml config file.
```

Comment:
Can you add some notes as to how to do this?

Reply:
I have added description about how to set up AccessKey pair.